### ISSUE TITLE
Jhn 10:34 ULB - space between quotation marks

### ISSUE THREAD

**Issue by lrsallee** [Door43/ulb-en#82]
The extra space between the " and the ' cause a ULB snippet mismatch. Why is it here?

**Comment by TomWarren** [comment]
Removed the space and ordered the three levels of quotation.
Thank you.

**Comment by hharrissWA** [comment]
Tom, is there a standard for not including a space between quotation marks?  My understanding has been that a space is needed, as I just described to Larry. What is our standard?
Thanks,
Harry

**Comment by TomWarren** [comment]
I would just do a search on the "'" or whatever, and see what is there. Usually, we have been dealing with these forms for months, and you can see what has been the pattern. When in doubt, don't. Just ask.

Better not to put something in at this stage than to create a new form that may be inconsistent.

It could be that these forms might change, but if "'" becomes " ' " that can be done with a script. Consistency is what saves time, and the aberrant or unusual forms take time to search them out.

I have been using the principle that most quotation marks, as spelling issues, should be left alone. We should be looking for obvious mistakes or omissions, I think. Everyone has their one preferred way of solving punctuation issues, but it is best, at this late stage not to be messing with quotations. If there are questions, just make a note in the report and send it in and Jim Pohlig and I will walk through them.

We are nearing the end, and we want to keep from introducing new ideas into the text. Just to save a ton of time, that is the wise way to go.

Thanks,

Tom

________________________________

**Comment by hharrissWA** [comment]
Thanks. I haven't been in doubt. I have been putting the space(s) in for well over a year. I would not be surprised if Jim isn't still doing that (but I wouldn't be surprised if he isn't either. [😊] )


I have no problem using the accepted form that everyone is using, but that is what I thought I was doing. I will not place any more spaces in the marks, going forward. I have edited a few with spaces, but I have also seen others with the spaces already there. So, as you describe, we will likely just need to go back through these with a script when we're done.


Thanks Tom,

Harry


C. Harry Harriss

Associate Editor, Unfolding Word Bible

Translation Consultant (in training)

Wycliffe Associates and Wycliffe India

harry_harriss@wycliffeassociates.org<mailto:harry_harriss@wycliffeassociates.org>

662-890-5637 home


________________________________

**Comment by TomWarren** [comment]
Sorry, Harry, they have not been in the text (as the norm)  for years. 
Dave Statezni has been removing those extra spaces (within the multiple 
quotations marks) that occurred in the chunk pages. And, just to say it 
again, we are not making any changes to the ULB or UDB at this late 
stage, without Jim Pohlig and me approving them.  This is to avoid 
changes that depart from what has been done and agreed-upon.  There are 
to be no extra spaces in the entire text. We bounced this around this 
morning with the Translation Notes and Text managers, and were agreed 
that there is to be no extra spaces  (between words, or after periods, 
or between quotation marks).

There were some who remembered that tA had the extra space in multiple 
quotation marks, others remembered that, for consistency, it was decided 
about a year ago that there were to be no extra spaces anywhere, with 
the ability to globally search and replace in the future, should there 
be editorial reasons and broad consensus for doing that. Jim Pohlig did 
not agree that he had used the spaces within quotation marks. Maybe 
someone else pushed for that, but no one owned it.

So, during the final edit, send in your Reader reports to me, and Jim 
and I will go over them, just to be sure that all changes would be given 
special oversight to ensure that we are not going backward.

Hope that helps. Sorry if I wasn't clear about this.

Tom


This is why I have been

On 6/13/2016 10:51 PM, hharrissWA wrote:
> Thanks. I haven't been in doubt. I have been putting the space(s) in 
> for well over a year. I would not be surprised if Jim isn't still 
> doing that (but I wouldn't be surprised if he isn't either. [😊] )
>
>
> I have no problem using the accepted form that everyone is using, but 
> that is what I thought I was doing. I will not place any more spaces 
> in the marks, going forward. I have edited a few with spaces, but I 
> have also seen others with the spaces already there. So, as you 
> describe, we will likely just need to go back through these with a 
> script when we're done.
>
>
> Thanks Tom,
>
> Harry
>
>
> C. Harry Harriss
>
> Associate Editor, Unfolding Word Bible
>
> Translation Consultant (in training)
>
> Wycliffe Associates and Wycliffe India
>
> harry_harriss@wycliffeassociates.org<mailto:harry_harriss@wycliffeassociates.org> 
>
>
> 662-890-5637 home
>
>
> ________________________________
> From: Tom Warren <notifications@github.com>
> Sent: Monday, June 13, 2016 9:37:15 PM
> To: Door43/ulb-en
> Cc: Harry Harriss; Comment
> Subject: Re: [Door43/ulb-en] Jhn 10:34 ULB - space between quotation 
> marks (#82)
>
> I would just do a search on the "'" or whatever, and see what is 
> there. Usually, we have been dealing with these forms for months, and 
> you can see what has been the pattern. When in doubt, don't. Just ask.
>
> Better not to put something in at this stage than to create a new form 
> that may be inconsistent.
>
> It could be that these forms might change, but if "'" becomes " ' " 
> that can be done with a script. Consistency is what saves time, and 
> the aberrant or unusual forms take time to search them out.
>
> I have been using the principle that most quotation marks, as spelling 
> issues, should be left alone. We should be looking for obvious 
> mistakes or omissions, I think. Everyone has their one preferred way 
> of solving punctuation issues, but it is best, at this late stage not 
> to be messing with quotations. If there are questions, just make a 
> note in the report and send it in and Jim Pohlig and I will walk 
> through them.
>
> We are nearing the end, and we want to keep from introducing new ideas 
> into the text. Just to save a ton of time, that is the wise way to go.
>
> Thanks,
>
> Tom
>
> ________________________________
> From: hharrissWA <notifications@github.com>
> Sent: Monday, June 13, 2016 9:27:28 PM
> To: Door43/ulb-en
> Cc: Tom Warren; State change
> Subject: Re: [Door43/ulb-en] Jhn 10:34 ULB - space between quotation 
> marks (#82)
>
>
> Tom, is there a standard for not including a space between quotation 
> marks? My understanding has been that a space is needed, as I just 
> described to Larry. What is our standard?
> Thanks,
> Harry
>
> -
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on 
> GitHub<https://github.com/Door43/ulb-en/issues/82#issuecomment-225755909>, 
> or mute the 
> thread<https://github.com/notifications/unsubscribe/AQ50nxiRVUmfOjoEAXROz-hlzQ30xQ5Bks5qLgOAgaJpZM4I03c0>.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on 
> GitHub<https://github.com/Door43/ulb-en/issues/82#issuecomment-225757236>, 
> or mute the 
> thread<https://github.com/notifications/unsubscribe/ARWui8-mfLcVNhXiIJaG6uVOBTqUojUpks5qLgXLgaJpZM4I03c0>.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Door43/ulb-en/issues/82#issuecomment-225766391>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AQ50n1tSXjfmW6hanxMVOD7WbP5qfhshks5qLhc8gaJpZM4I03c0>.
>



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus

**Comment by hharrissWA** [comment]
No problem, Tom. I just needed clarity. I suppose going from Text to Notes Writing to Translation Academy to the Son of God Notes and back to Text has caused me to miss some of these decisions. I have no problem with however it is to be done. I just need to know. [😊]  Perhaps a list of common rules for these sorts of items would be helpful?


I am unclear about sending in my Reader Reports. Are you referring to the Reader Reports that have been sent to me and which I completed, or to my final editing in Mark (and now 2 Corinthians in which I am working)?


In Mark I have made a list of questions I have for the Text Team about some specific issues (which I have sent to you), but I have not made a personal log of every small edit I have made because those are recorded in Github. If there is a specific item that someone needs to go back and look at, I can do that, for example, going back and checking all the quotation marks and deleting spaces (if running a script is not able to fix that?).


In other words, I just need to know specifically how the text should look in regard to punctuation, and I will follow that and I will also be glad to go back and check whatever is needed and make corrections.


Thanks,

Harry


C. Harry Harriss

Associate Editor, Unfolding Word Bible

Translation Consultant (in training)

Wycliffe Associates and Wycliffe India

harry_harriss@wycliffeassociates.org<mailto:harry_harriss@wycliffeassociates.org>

662-890-5637 home


________________________________
